src/main/java/com/github/osisoft/ocs_sample_library_preview/OCSClient.java

```diff
@@ -85,4 +85,23 @@ private void init() {
         return mGson.fromJson(input, new TypeToken<Map<String, Object>[]>() {
         }.getType());
     }
+
+    /**
+     * Helper function to get standard request headers
+     * 
+     * @return Map<String,String> object with http headers
+     */
+    public Map<String, String> getHttpHeadersForRequest() {
+        return baseClient.getHttpHeadersForRequest();
```

Comment:
I find it interesting that the OCSClient doesn't just extend the BaseClient. I am not suggesting that we change that, but I find it a little odd.

Reply:
Agreed. I wouldn't be opposed to changing that behavior in a separate WI.